pkg/config/setup/system_probe.go

```diff
@@ -400,6 +400,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 
 	// Discovery config
 	cfg.BindEnvAndSetDefault(join(discoveryNS, "enabled"), false)
+	cfg.BindEnvAndSetDefault(join(discoveryNS, "cpu_usage_update_delay"), "60s")
```

Comment:
should the configuration be provided by the datadog configuration and not by the check configuration?